sdk-python/create_bulk_messages.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+import os
+import uuid
+
+from awaazde import AwaazDeAPI
+from awaazde.constants import CommonConstants
+from awaazde.utils import CSVUtils
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("username", help="Username", type=str)
+    parser.add_argument("password", help="Password", type=str)
+    parser.add_argument("organization", help="Organization ", type=str)
+    parser.add_argument("csv_file_path", help="Path for the csv ", type=str)
```

Comment:
this command is called create_bulk_messages, so this var sh be named accordingly. How about `messages_file_path`?

And correspondingly, rename the command to `create_messages`?